types/hapi__hapi/test/server/server-cache.ts

```diff
@@ -1,6 +1,6 @@
 // https://github.com/hapijs/hapi/blob/master/API.md#-servercacheoptions
 import { Server, ServerOptionsCache } from "@hapi/hapi";
-import * as catbox from "@hapi/catbox";
+import { PolicyOptions } from '@hapi/catbox';
```

Comment:
```suggestion
import { PolicyOptions } from "@hapi/catbox";
```

Either this should be double-quoted or the other import should be single-quoted I guess?

Reply:
It's something my IDE did, I will see if I can consolidate if the header is the only inconsistency but it's out of scope of this PR.